apps/nextjs-website/src/components/organisms/GuideInPageMenu/GuideInPageMenu.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode } from 'react';
+import { parseInPageMenu } from 'gitbook-docs/parseInPageMenu';
+import {
+  RenderingComponents,
+  renderInPageMenu,
+} from 'gitbook-docs/renderInPageMenu';
+import Heading from '@/components/organisms/GuideInPageMenu/components/Heading';
+
+type GuideInPageMenuProps = {
+  assetsPrefix: string;
+  pagePath: string;
+  inPageMenu: string;
+};
+
+const components: RenderingComponents<ReactNode> = {
+  Heading: Heading,
+};
+
+const GuideInPageMenu = ({
+  inPageMenu,
+  assetsPrefix,
+  pagePath,
+}: GuideInPageMenuProps) => {
+  const nodes = parseInPageMenu(inPageMenu, { assetsPrefix, pagePath });
+
+  // Do not render the menu if it contains only the title
+  return nodes.length > 1 ? renderInPageMenu(nodes, React, components) : null;
```

Comment:
Is this still correct after the changes on [parseInPageMenu](https://github.com/pagopa/developer-portal/pull/230/files#diff-a98e901a8f343a8ae8fef29a84e6682f683304ca9918b4e2d4252636db965726R26) function

Reply:
Nope